changelog-preview.sh

```diff
@@ -7,7 +7,20 @@ function _changelogsh_preview {
     raw_version=$1
   fi
 
-  if [ $raw_version != "Unreleased" ]; then
+  if [ $raw_version == "-all" ]; then
```

Comment:
```suggestion
  if [ $raw_version == "--all" ]; then
```
single dash is normally for one letter shortcuts like `-r` for `--recursive` and `-d` for `--daemon`. double dash is more appropriate here for `--all`